java/compaction/compaction-status-store/src/test/java/sleeper/compaction/status/store/job/QueryCompactionJobStatusUnfinishedIT.java

```diff
@@ -38,23 +40,40 @@ public void shouldReturnUnfinishedCompactionJobs() {
         // Given
         Partition partition = singlePartition();
         FileReferenceFactory fileFactory = fileFactory(partition);
-        CompactionJob job1 = jobFactory.createCompactionJob(
+        CompactionJob job = jobFactory.createCompactionJob(
                 List.of(fileFactory.rootFile("file1", 123L)),
                 partition.getId());
```

Comment:
The name of this test suggests it should return more than one compaction job. It looks like there's no test that does that now. Please can we revert it?